PYME/Acquire/Hardware/Simulator/rend_im.py

```diff
@@ -303,9 +302,9 @@ def interp3(X, Y, Z):
 @fluor.registerIllumFcn
 def PSFIllumFunction(fluors, position):
     im = interpModel()
-    xi = maximum(minimum(round_((fluors['x'] - position[0])/dx + im.shape[0]/2).astype('i'), im.shape[0]-1), 0)
```

Comment:
why `round_()` here (and is this the same as `np.round_()` and not some overridden round that I wrote somewhere)

Reply:
Not some overridden one you wrote unless you wrote - only potential would be in `PYME/Analysis/PSFGen/ps_app.c` and I checked there.

This falls under me trying not to change style things/whitespace - they're the same:
```
>>> np.round
<function round_ at 0x7fce619d92f0>
```